### PR TITLE
CI: fix false failing in `dashboard_spec.js`

### PR DESCRIPTION
**Pull request by naspeh**
Another attempt to catch this up.

**Updated:**
Second commit is just a little performance improvement (less page reloadings)

### CODE REVIEW

**Comment by petrjasek** [comment]
where is the picture?

**Comment by naspeh** [comment]
https://test.superdesk.org/logs/screenshots/sdcpr-1840/

**Comment by petrjasek** [comment]
nice :+1:

**Comment by naspeh** [comment]
yes, but test is still failing %)... I don't understand why...

File: spec/dashboard_spec.js [hunk 1]
```diff
@@ -44,12 +44,15 @@ describe('dashboard', () => {
...
-            5000
-        );
+
+        // TODO
+        // this second saving is needed for passing tests on CI,
+        // I could reproduce this scenario locally once, so probably
+        // there is can be issue with first saving...
+        // Same thing was in line 74, so probably someone got this before too
+        dashboard.showMonitoringSettings(0);
+        monitoring.saveSettings();
```
Comment (petrjasek) [inline spec/dashboard_spec.js]: just wondering if this is still needed with nav?

Comment (naspeh) [reply spec/dashboard_spec.js]: I think so, when I got this scenario locally once I didn't refresh the page...

I'll check once again on Monday.

Comment (naspeh) [reply spec/dashboard_spec.js]: I've just checked and this's still needed.

**Pull request merged**